cactusbot/handler.py

```diff
@@ -13,6 +13,10 @@ def __init__(self, *handlers):
 
         self.handlers = handlers
 
+    async def register_handler(self, handler):
```

Comment:
I forgot to remove that, it's not actually being used. I thought of that while I was writing that, and changed it to an event.